app/src/atoms/AtomLink.re

```diff
@@ -1,12 +1,23 @@
 open Router;
+open Css;
+
+module Styles = {
+  let link = style([]);
+};
 
 [@react.component]
-let make = (~page: page, ~children=?) => {
+let make = (~page: page, ~style=?, ~children=?) => {
   let content =
     switch (children) {
     | Some(child) => child
     | None => React.string(nameOfPage(page))
     };
 
-  <a onClick={navigateOfPage(page)}> content </a>;
+  let style =
+    switch (style) {
+    | Some(style) => merge([Styles.link, style])
```

Comment:
Just for my own clarification, is the order of styles passed to merge significant?

Reply:
Yes, it is significant. The latter overrides the former if they contain the same attributes.

My thought behind this is that a component has a default style which can be made more specific (or altered) from outside. This way we'll have to make less variations of our basic elements.

Reply:
Good to know the presidence for that. Thanks!